.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "prettier.prettierPath": "./node_modules/prettier",
```

Comment:
Apparently the vscode plugin for prettier v3 now automatically finds the prettier package. It gave me some `TypeError` until I found this fix on their repo.